### PR TITLE
add router, add singin page

### PR DESCRIPTION
**Pull request by uvolovik**

### CODE REVIEW

File: src/SignIn/Sign.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2)
+  },
+  container: {
+    position: "absolute",
+    top: "45%",
+    left: "50%",
+    marginRight: "-50%",
+    transform: "translate(-50%, -50%)"
```
Comment (ivanprotasov) [inline src/SignIn/Sign.tsx]: height: '100vh', display:flex

File: src/SignIn/Sign.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  }
+}));
+
+export default function SignIn() {
+  const classes = useStyles();
+
+  return (
+    <Container className={classes.container} component="main" maxWidth="xs">
+      <CssBaseline />
+      <div className={classes.paper}>
```
Comment (ivanprotasov) [inline src/SignIn/Sign.tsx]: margin: auto
Not the best class name

File: src/SignIn/Sign.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        <Avatar className={classes.avatar}>
+          <FreeBreakfast />
+        </Avatar>
+        <Typography component="h1" variant="h5">
+          Sign in
+        </Typography>
+        <form className={classes.form} noValidate>
+          <TextField
```
Comment (ivanprotasov) [inline src/SignIn/Sign.tsx]: Use Input please

File: src/components/Profile/Profile.tsx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    milk: true,
+    start: "09:00",
+    end: "18:00"
+  });
+  return (
+    <div className={classes.root}>
+      <CssBaseline />
+      <Header />
+      <main className={classes.content}>
+        color: '#fff',
```
Comment (ivanprotasov) [inline src/components/Profile/Profile.tsx]: what is this? :)

File: src/components/Profile/Profile.tsx [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+        <div className={classes.toolbar} />
```
Comment (ivanprotasov) [inline src/components/Profile/Profile.tsx]: toolbarPlaceholder

File: src/components/Profile/Profile.tsx [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+                    name="name"
+                    onChange={handleInputChange}
+                  />
+                  {/* <FormHelperText id="component-error-text">Error</FormHelperText> */}
+                </FormControl>
+                <FormControl className={classes.formControl}>
+                  <InputLabel htmlFor="component-simple">Email</InputLabel>
+                  <Input
+                    type="email"
+                    id="component-simple"
```
Comment (ivanprotasov) [inline src/components/Profile/Profile.tsx]: the same IDs

File: src/components/Profile/Profile.tsx [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+                </FormControl>
+                <FormControl className={classes.formControl}>
+                  <InputLabel htmlFor="component-simple">Location</InputLabel>
+                  <Input
+                    id="component-simple"
+                    value={values.location}
+                    name="location"
+                    onChange={handleInputChange}
+                  />
+                  {/* <FormHelperText id="component-error-text">Error</FormHelperText> */}
```
Comment (ivanprotasov) [inline src/components/Profile/Profile.tsx]: remove copy paste

File: src/components/Profile/Profile.tsx [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+                      onChange={handleInputChange}
+                    />
+                  </FormControl>
+                </div>
+              </form>
+            </Paper>
+          </Grid>
+        </Grid>
+      </main>
+    </div>
```
Comment (ivanprotasov) [inline src/components/Profile/Profile.tsx]: add submit button

File: src/components/Header/Header.scss [hunk 9]
```diff
@@ -3,3 +3,7 @@
   display: flex;
   justify-content: space-between;
 }
+
+.logout-button{
+  text-decoration: none;
+}
```
Comment (tatyanamatesha) [inline src/components/Header/Header.scss]: please, add new line.

File: src/components/Header/Header.tsx [hunk 10]
```diff
@@ -31,9 +32,15 @@ export default function Header() {
             ☕ Coffee
           </Typography>
 
-          <Button variant="outlined" color="default" startIcon={<ExitToApp />}>
-            LOGOUT
-          </Button>
+          <Link to="/login" className="logout-button">
+            <Button
```
Comment (tatyanamatesha) [inline src/components/Header/Header.tsx:20]: please, add `aria-label` in order to clarify `Button` purpose for anyone who cannot access the visual indication.

File: src/pages/Profile/Profile.tsx [hunk 11]
```diff
@@ -0,0 +1,206 @@
...
+  };
+
+  interface IForm {
+      name: string,
+      email: string,
+      skype: string,
+      location: string,
+      milk : boolean,
+      start: string,
+      end: string,
```
Comment (tatyanamatesha) [inline src/pages/Profile/Profile.tsx]: please, use 2 spaces.

File: src/pages/Profile/Profile.tsx [hunk 12]
```diff
@@ -0,0 +1,206 @@
...
+                      }}
+                      onChange={handleInputChange}
+                    />
+                  </FormControl>
+                </div>
+              </form>
+            </Paper>
+          </Grid>
+        </Grid>
+      </main>
```
Comment (tatyanamatesha) [inline src/pages/Profile/Profile.tsx]: can we create separate component for this? hard to read code. maybe we can reduce this with 2 mappers? what do you think?

**Review by tatyanamatesha — CHANGES_REQUESTED** [review]

**Review by ivanprotasov — APPROVED** [review]

File: src/components/Header/Header.tsx (hunk 10 above)
Comment (ivanprotasov) [reply src/components/Header/Header.tsx:20]: we will not support accessibility

**Review by tatyanamatesha — APPROVED** [review]

**Pull request merged**